src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.tsx

```diff
@@ -68,43 +43,26 @@ export const AssetDataListItemStructure: React.FC<IAssetDataListItemStructurePro
         fallback: '-',
     });
 
-    const formattedPriceChanged = formatterUtils.formatNumber(fiatAmountChanged, {
-        format: NumberFormat.FIAT_TOTAL_SHORT,
-        withSign: true,
-    });
-
-    const formattedPriceChangedPercentage = formatterUtils.formatNumber(priceChange / 100, {
-        format: NumberFormat.PERCENTAGE_LONG,
-        withSign: true,
-    });
-
     return (
-        <DataList.Item className={classNames('flex items-center gap-x-3 py-3 md:py-5', className)} {...otherProps}>
-            <Avatar src={logoSrc} responsiveSize={{ md: 'md', sm: 'sm' }} className="block" />
-            <div className="flex w-full min-w-0 shrink items-center justify-between gap-3">
-                <div className="flex flex-col gap-y-1 truncate">
-                    <span className="truncate text-base leading-tight text-neutral-800 md:text-lg">{name}</span>
-                    <p className="truncate text-sm leading-tight text-neutral-500 md:text-base">
-                        <span>{formattedAmount} </span>
-                        <span className="truncate">{symbol}</span>
-                    </p>
-                </div>
-                <div className="flex flex-col items-end justify-center gap-y-1">
-                    {fiatPrice ? (
-                        <>
-                            <span className="text-base leading-tight text-neutral-800 md:text-lg">
-                                {formattedPrice}
-                            </span>
-                            <div className="flex items-center gap-x-1">
-                                <span className={changedAmountClasses}>{formattedPriceChanged}</span>
-                                <Tag label={formattedPriceChangedPercentage!} variant={tagVariant} />
-                            </div>
-                        </>
-                    ) : (
-                        <span className="text-sm leading-tight text-neutral-800 md:text-base">
-                            {copy.assetDataListItemStructure.unknown}
-                        </span>
-                    )}
+        <DataList.Item
+            className={classNames('flex items-center justify-between gap-x-3 py-3 md:py-5', className)}
+            {...otherProps}
+        >
+            <div className="flex min-w-0 items-center gap-3">
+                <Avatar src={logoSrc} responsiveSize={{ md: 'md', sm: 'sm' }} className="block shrink-0" />
+                <span className="truncate text-base leading-tight text-neutral-800 md:text-lg">{name}</span>
+            </div>
+            <div className="flex min-w-0 items-end justify-end gap-x-2 text-right">
+                <div className="flex min-w-0 flex-col gap-y-1">
+                    <span className="shrink-0 text-base leading-tight text-neutral-800 md:text-lg">
+                        {fiatPrice ? formattedPrice : copy.assetDataListItemStructure.unknown}
```

Comment:
The formatter utils supports a fallback string, we could specify the `unknown` string directly there

Reply:
@cgero-eth I actually tried this first, but it didn't work, I guess because we default fiat amount  to 0, but we can safely remove that actually as it will just return NaN in the fiatAmount var and use the fallback